omod/src/main/java/org/openmrs/web/attribute/handler/SerializingFieldGenDatatypeHandler.java

```diff
@@ -33,6 +44,33 @@ public T getValue(DT datatype, HttpServletRequest request, String formFieldName)
 		if (StringUtils.isBlank(formFieldValue)) {
 			return null;
 		}
+		if (StringUtils.isNumeric(formFieldValue)) {
+			BaseOpenmrsObject metadata = null;
+			PropertyEditor editor;
+
```

Comment:
Can you add some unit tests for these?

Reply:
@dkayiwa Done :) I've managed to get 100% coverage